static/app/views/replays/detail/network/index.tsx

```diff
@@ -123,7 +135,7 @@ const Item = styled('div')<{color?: string; numeric?: boolean}>`
   display: flex;
   align-items: center;
   max-height: 28px;
-  color: ${p => p.color || p.theme.subText};
+  color: ${({theme, color}) => (color ? theme[color] || theme.subText : theme.subText)};
```

Comment:
This looks a little strange to me is there a reason you couldn't just do `color ? theme[color] : theme.subText` or `theme[color] ?? theme.subText`